### PR TITLE
added giang info (2)

### PR DESCRIPTION
**Pull request by ghpham25**
a Khoa check cho em lần này được chưa ạ? e mò mãi mới ra TT

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -407,5 +407,19 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+	{
+		name: 'Giang Pham',
+		title: 'Student @ Colby College',
+		hobbies: 'Kpop, cooking, japanese books',
+		avatar:
+			'https://scontent.fman2-1.fna.fbcdn.net/v/t39.30808-6/269677226_1349384122150058_5399863847506458060_n.jpg?_nc_cat=108&ccb=1-7&_nc_sid=5f2048&_nc_ohc=A6VibfuVTWIAX9CmDUi&_nc_ht=scontent.fman2-1.fna&oh=00_AfDk68mpuR07R8PHvbhtIb7vEJIDU0W6zJYT9Le7Bv3-mg&oe=6603B1B2',
```
Comment (khoa165) [inline index.js]: let's change to this URL  `https://res.cloudinary.com/khoa165/image/upload/v1711262026/viettech/giang.jpg`

**Comment by ghpham25** [comment]
I updated my profile pic to the new URL

**Review by khoa165 — APPROVED** [review]

**Comment by khoa165** [comment]
@ghpham25 you have git conflict, might wanna practice resolving this

**Comment by ghpham25** [comment]
I had some trouble solving the conflict on my terminal. 

when I ran `git merge main` on my giangchange2 branch, the terminal showed "up to date". I tried `git pull origin giangchange2` and `git pull origin main`, but both showed up to date, so even though I could see the conflict on GitHub, it didn't show on my terminal. so I ended up just resolving the conflict on GitHub. 
 
If there is anything I did wrong?

**Comment by khoa165** [comment]
@ghpham25, it does look like your conflict went away, so i am gonna merge this

**Pull request merged**